schemas/postgres.sql

```diff
@@ -53,9 +54,8 @@ CREATE TABLE IF NOT EXISTS characteristic_ratings (
 );
 
 CREATE TABLE IF NOT EXISTS characteristics (
-  characteristic_id INT GENERATED ALWAYS AS IDENTITY,
+  id INT GENERATED ALWAYS AS IDENTITY,
+  product_id INT,
   value TEXT,
-  PRIMARY KEY(characteristic_id)
+  PRIMARY KEY(id)
 );
-
-SELECT * FROM characteristics;
```

Comment:
Is this statement used in any server method/script or is it just to be called from the command line once for creation?

Reply:
I was just using it for testing and forgot to remove it